src/share/select.js

```diff
@@ -44,7 +53,7 @@ export default function Select(
         onChange={onChange}
         size={size}
         value={value}
-        disabled={disabled}
+        disabled={disabled || false}
```

Comment:
disabled would just be undefined.  Which is the same I guess.  We might not need that there.